client/my-sites/theme/main.jsx

```diff
@@ -715,6 +719,32 @@ class ThemeSheet extends Component {
 		);
 	};
 
+	renderThemeBadge = () => {
+		const { siteId, siteSlug, softLaunched, themeId, themeTier, themeType } = this.props;
+
+		const isCommunityTheme = themeType === DOT_ORG_THEME;
+		const isPartnerTheme = themeTier.slug === 'partner';
+		const isSenseiOrWooCommerceTheme = themeType === BUNDLED_THEME;
+
+		if ( ! isCommunityTheme && ! isPartnerTheme && ! isSenseiOrWooCommerceTheme ) {
+			return null;
+		}
+
+		const className = classNames( 'theme__sheet-main-info-type', {
+			'is-soft-launched': softLaunched,
+		} );
+		return config.isEnabled( 'themes/tiers' ) ? (
```

Comment:
I think we could make the code more readable by changing this ternary to an explicit if.

````
if ( config.isEnabled( 'themes/tiers' ) ) {
    return ( A )
}

return ( B )
````
Being A and B the different components.

Reply:
Could this be a matter of personal preference? I find that a single `return` statement with two simple blocks is more readable than code nested inside an `if` and then another `return`.

I checked the coding guidelines to see if they mentioned ternary operators but didn't find anything.

Reply:
Sure, leave the ternary then.